apps/frontend/src/libs/components/link/link.tsx

```diff
@@ -5,11 +5,18 @@ import { type ValueOf } from "~/libs/types/types.js";
 
 type Properties = {
 	children: React.ReactNode;
+	className?: string;
```

Comment:
```suggestion
```
Make it customizable using props instead

Reply:
I don't quite understand what that means. I need to add for example prop type and use like this?
```
type Properties = {
	type: string;
}

	<NavLink className={clsx(styles["link"], type)} to={to}>

```

Reply:
Yes, this approach is better. But be more specific with the type of `type` variable, not just string